src/main/web_src/src/pages/gruppeOversikt/FinnPerson.tsx

```diff
@@ -88,46 +76,25 @@ export default function FinnPerson({ naviger }: FinnPerson) {
 	if (redirectToGruppe) return <Redirect to={`/gruppe/${gruppe}`} />
 
 	return (
-		<>
-			{/* @ts-ignore */}
-			<Søkeknapp onClick={openFinnModal} style={{ marginTop: '10px' }} kompakt="">
-				<span>Finn testperson</span>
-			</Søkeknapp>
-
-			<DollyModal isOpen={isFinnModalOpen} closeModal={handleClose} width="40%" overflow="visible">
-				<h1>Søk etter testperson</h1>
-				<p>
-					Du kan søke på både FNR/DNR/BOST og navn, eller deler av disse. Du vil da få en liste av
-					aktuelle personer. Velg en person for å bli tatt direkte til testdatagruppen personen
-					finnes i.
-				</p>
-				{/* @ts-ignore */}
-				<Label name="Testperson" label="Testperson">
-					<AsyncSelect
-						defaultOptions={false}
-						loadOptions={fetchOptions}
-						onInputChange={handleChange}
-						options={options}
-						onChange={(e: Option) => setIdent(e.value)}
-						cacheOptions={true}
-						label="Person"
-						placeholder="Begynn å skrive inn FNR/DNR/BOST eller navn..."
-					/>
-				</Label>
-				{feilmelding && (
-					<div className="error-message" style={{ marginBottom: '10px' }}>
-						{feilmelding}
-					</div>
-				)}
-				<div className="flexbox--all-center" style={{ marginTop: '20px' }}>
-					<NavButton type="standard" onClick={handleClose} style={{ marginRight: '10px' }}>
-						Avbryt
-					</NavButton>
-					<NavButton type="hoved" onClick={navigerTilPerson}>
-						Gå til person
-					</NavButton>
+		<div>
+			<div className="finnperson-container skjemaelement">
+				<AsyncSelect
```

Comment:
Synes det var litt vanskelig å skjønne at dette var et søkefelt, for det er liksom ikke noe her som forteller meg hva det er 😉  Kunne vi fått med et sånn forstørrelsesglass-ikon? Evt. starte placeholder-tekst med "Søk etter..."? Eller kanskje aller best - begge deler? 😁

Reply:
Det er sant, litt uoversiktlig nå ser jeg 😂  Begge deler høres supert ut, skal fikse 😄